packages/ui_v2/src/components/commonV2/FormField/index.tsx

```diff
@@ -140,7 +140,7 @@ const StringFieldView = observer(
             <span>{field.description}</span>
           </MDBTooltip>
           <input
-            type="text"
+            type={(field.displayName === "Threshold Const" ||  field.displayName === "DAO Bounty Const") ? "number" : "text"}
```

Comment:
This file is a generic "Form Field Drawer", why are we adding specific display name conditions here? This feels very wrong.

Reply:
I thought the same, but we are using this same input (StringField) to add the name and token of organization; so this was the way to achieve the number input in the advance form. Maybe i’m missing something?

Reply:
Also, I checked v1 and we used this
```
InputProps={{
          startAdornment: FieldInformation(popupState)
        }}
```
Which is something of Material UI (I think that define the input type), so I decided to do it this way. Let me know if it makes sense please!